manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0

```

```suggestion
PackageVersion: 1.8
```
This must match Control Panel.